### PR TITLE
Initial queries

### PR DESCRIPTION
**Pull request by westonio**
#### Co Authors:
@ChrisAsercion @Gabe-Torres @z-fitch 

### Description of changes
GraphQL features:
- Add Reviews query to return all Review object
- Add Review(id) query to return a specified object
- Add createReview mutation to create review objects

### Test Suite
_NOTE: There are currently no tests for these implementations. I can commit to creating these tests over the weekend_

#### Add GIF (REQUIRED)
giphy

### CODE REVIEW

File: db/seeds.rb
Comment (Gabe-Torres) [inline db/seeds.rb:1]: Thanks for creating this seed data! Should come in handy.

**Review by ChrisAsercion — APPROVED** [review]
We worked on this together as a group and it looks solid. Kinks were worked out for now.

**Review by z-fitch — APPROVED** [review]
This looks good, but we need to remember (later) to resolve the comment todo

File: app/graphql/mutations/create_review.rb
Comment (Gabe-Torres) [inline app/graphql/mutations/create_review.rb:1]: Awesome job with the mutation setup! Now we'll be able to create and change data.

**Review by Gabe-Torres — APPROVED** [review]

**Pull request merged**